### DISCUSSION TITLE
Build #1

### DISCUSSION THREAD

**Discussion by Morta1** [Morta1/IdleonToolbox#123] [Builds]
Build

**Comment by Shobon01** [comment]
{"tabs":[{"index":0,"name":"Rage_Basics","note":">1< If u have some point\n>2< If u have some point from >1<","talents":[{"name":"Health_Booster","skillIndex":0,"level":0,"note":"","description":"Increases_Max_HP_by_{","x1":1,"x2":0.15,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{_HP"},{"name":"Mana_Booster","skillIndex":1,"level":0,"note":"","description":"Increases_Max_MP_by_{","x1":1,"x2":0.1,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{_MP"},{"name":"Star_Player","skillIndex":8,"level":0,"note":"","description":"Gives_{_Star_Talent_Points._Star_Talents_are_found_throughout_the_game!","x1":1,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+1_Star_Talent_Point"},{"name":"Bucklered_Up","skillIndex":9,"level":0,"note":"","description":"Increases_Total_Defense_by_+{%","x1":40,"x2":60,"funcX":"decay","y1":null,"y2":null,"funcY":"_","lvlUpText":"+{%_Total_DEF"},{"name":"Sharpened_Axe","skillIndex":5,"level":0,"note":"","description":"Increases_Base_Weapon_Power_by_{._This_contributes_to_your_total_damage.","x1":0.25,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"_","lvlUpText":"+{_Base_Weapon_Power"},{"name":"Fist_Of_Rage","skillIndex":10,"level":">2<","note":"","description":"Increases_your_Base_STR_by_{_(STR_boosts_Max_HP_and_Crit_Damage)","x1":1,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{_STR"},{"name":"Quickness_Boots","skillIndex":11,"level":0,"note":"","description":"Increases_your_Base_AGI_by_{_(AGI_boosts_Movement_Speed_and_Crit_Chance)","x1":1,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{_AGI"},{"name":"Book_Of_The_Wise","skillIndex":12,"level":0,"note":"","description":"Increases_your_Base_WIS_by_{_(WIS_boosts_Mana_and_Boss_Damage)","x1":1,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{_WIS"},{"name":"Lucky_Clover","skillIndex":13,"level":0,"note":"","description":"Increases_your_Base_LUK_by_{_(LUK_boosts_drop_rate,_EXP_Gain,_and_other_RNG)","x1":1,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{_LUK"},{"name":"Gilded_Sword","skillIndex":6,"level":0,"note":"","description":"Boosts_Damage_dealt|to_all_monsters_by_{%","x1":1,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"_","lvlUpText":"+{%_Damage"},{"name":"Brute_Efficiency","skillIndex":85,"level":"100","note":"","description":"Increases_the_total_efficiency_of_all_specialized_skills_by_{%","x1":1,"x2":0.02,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_Efficiency"},{"name":"Meat_Shank","skillIndex":86,"level":0,"note":"","description":"Damage_dealt_is_increased_by_{%_for_every_power_of_10_Max_HP_you_have","x1":100,"x2":80,"funcX":"decay","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_Damage"},{"name":"Critikill","skillIndex":87,"level":0,"note":"","description":"Increases_Critical_Hit_Damage_by_{%","x1":52,"x2":50,"funcX":"decay","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_Critical_Damage"},{"name":"Idle_Brawling","skillIndex":88,"level":0,"note":"","description":"AFK_Gains_Rate_for_Fighting_is_increased_by_{%","x1":20,"x2":50,"funcX":"decay","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_AFK_Gains_Rate"},{"name":"Idle_Skilling","skillIndex":89,"level":">1<","note":"","description":"AFK_Gains_Rate_for_all_Specialized_Skills_is_increased_by_{%","x1":20,"x2":40,"funcX":"decay","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_AFK_Gains_Rate"}]},{"index":1,"name":"Warrior","note":"1 = Put 1 point\n>1< If u have some point\n>2< If u have some point from 1","talents":[{"name":"Power_Strike","skillIndex":90,"level":0,"note":"","description":"Slash_forward_dealing_{%|damage_to_up_to|2_monsters","x1":130,"x2":3,"funcX":"bigBase","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_Damage","K":1.1,"D":1.17,"s":1.1,"cooldown":4,"castTime":0.45,"manaCost":5,"inputReq":0,"AFKrange":80,"AFKtype":"line","AFKactivity":0},{"name":"Whirl","skillIndex":91,"level":0,"note":"","description":"Swing_your_weapon_around_you|dealing_{%_damage_to_up_to|}_monsters","x1":60,"x2":1.5,"funcX":"bigBase","y1":3,"y2":24,"funcY":"intervalAdd","lvlUpText":"+{%_Damage_&_+}_Mobs_Hit","K":1.1,"D":1.18,"s":1.2,"cooldown":6,"castTime":0.45,"manaCost":9,"inputReq":0,"AFKrange":45,"AFKtype":"line","AFKactivity":0},{"name":"Health_Overdrive","skillIndex":92,"level":0,"note":"","description":"Increase_max_HP_by_{%","x1":0.5,"x2":0.02,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_HP"},{"name":"Double_Strike","skillIndex":93,"level":0,"note":"","description":"Basic_Attacks_with_Warrior_Weapons_have_a_{%_chance_to_hit_twice","x1":110,"x2":50,"funcX":"decay","y1":null,"y2":null,"funcY":"txt","lvlUpText":"{%_Double_Hit_Chance"},{"name":"Firmly_Grasp_It","skillIndex":94,"level":"1","note":"","description":"Temporarily_boosts_base_STR_by_{_for_}_minutes","x1":15,"x2":1,"funcX":"bigBase","y1":2,"y2":4,"funcY":"intervalAdd","lvlUpText":"+{_STR_&_+}_Mins","K":1,"D":1,"s":1,"cooldown":30,"castTime":1.5,"manaCost":5,"inputReq":0,"AFKrange":1,"AFKtype":"buff","AFKactivity":-1},{"name":"Strength_In_Numbers","skillIndex":95,"level":0,"note":"","description":"STR's_effect_on_both_Damage_and_HP_is_increased_by_{%","x1":0.75,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_Effect"},{"name":"'str'ess_Tested_Garb","skillIndex":96,"level":0,"note":"","description":"All_Equipment_gives_{%_more_STR_than_what's_listed","x1":1.5,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_more_STR_from_equips"},{"name":"Carry_A_Big_Stick","skillIndex":97,"level":0,"note":"","description":"The_effect_Weapon_Power_has_on_Damage_Dealt_is_increased_by_{%","x1":1,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_Weapon_Power_Effect"},{"name":"Absolute_Unit","skillIndex":98,"level":0,"note":"","description":"+{_base_STR","x1":1,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{_STR"},{"name":"Haungry_For_Gold","skillIndex":99,"level":0,"note":"","description":"Golden_Food_bonuses_are_increased_by_{%","x1":55,"x2":80,"funcX":"decay","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_Bonus"},{"name":"Big_Pick","skillIndex":100,"level":"100","note":"","description":"Swings_forward_and_mines_the_rock_with_+{%_bonus_Mining_Efficiency","x1":150,"x2":15,"funcX":"bigBase","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_Efficiency","K":1.05,"D":1.4,"s":1,"cooldown":12,"castTime":1.5,"manaCost":10,"inputReq":0,"AFKrange":0,"AFKtype":"buff","AFKactivity":1},{"name":"Copper_Collector","skillIndex":101,"level":">2<","note":"","description":"Mining_Efficiency_is_increased_by_+{%_for_every_power_of_10_Copper_Ore_in_Storage_Chest.","x1":20,"x2":70,"funcX":"decay","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_efficiency"},{"name":"Motherlode_Miner","skillIndex":102,"level":">1<","note":"","description":"+{%_base_multi-ore_drop_chance._This_can_trigger_up_to_4_times_in_a_row_per_swing.","x1":20,"x2":50,"funcX":"decay","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_Multi-Ore_Base_Chance"},{"name":"Tool_Proficiency","skillIndex":103,"level":"1","note":"","description":"Pickaxes_give_+{%_more_Mining_Power_than_whats_listed_for_every_10_Mining_Lv_you_have","x1":16,"x2":40,"funcX":"decay","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_POW_per_10_Mining_Lv"},{"name":"Tempestuous_Emotions","skillIndex":104,"level":0,"note":"","description":"Increases_Exp_Gain_for_all_Specialized_Skills_by_{%","x1":1,"x2":0,"funcX":"add","y1":null,"y2":null,"funcY":"txt","lvlUpText":"+{%_Exp_Gain"}]}],"class":"Warrior"}
